### ISSUE TITLE
Newsletter - neviditelné logo při black theme v outlooku

### ISSUE THREAD

**Issue by protivinsky** [faktaoklimatu/web-cz#962]
Newsletter se mi zobrazuje takto - viz níže - nevidím logo. Text se automaticky překlopí do bílé, ale logo ne. Nevím jestli má smysl to řešit, možná by stačilo jen nastavit white background (a taky nevím, kde je nejlepší místo na toto issue).

![image](https://user-images.githubusercontent.com/9573993/142585326-fcfebd1d-545a-4beb-bd09-c81017358033.png)

**Comment by mukrop** [comment]
Ano, s dark mode na mobile to riesim tiez a mam to v hlave ako drobne issue.

Za mna idealne riesenie je dat tam logo, ktore bude mat biely okraj na tom
ciernom pisme, alebo tam bude logo s obdlznikovym bielym pozadim (co je
vizualne slabsie, ale funkcne riesenie).
Ci si mysli @kikina? Slo by logo s transparentnym pozadim, ktore je na
tmavom podklade tiez citatelne?

(PS: Dat taketo issue do web-cz je asi ok.)

**Comment by mukrop** [comment]
Toto uz je myslim vyriesene? Ak ano, @tomasp-gr mozes issue zavriet.
